docs/UserGuide/Integration-Test/Integration-Test-refactoring-tutorial.md

```diff
@@ -220,44 +220,6 @@ public class IoTDBAlignedSeriesQueryIT {
 }
 ```
 
-### Commands for starting IT 
```

Comment:
why remove this?

Reply:
The commands part is already described at this link https://github.com/apache/iotdb/blob/master/integration-test/README.md

Reply:
Maybe we can put a link here. 
What do you think?

Reply:
+1 for a link